stdlib/public/stubs/LibcShims.cpp

```diff
@@ -15,10 +15,11 @@
 #include <cmath>
 #if defined(_WIN32)
 #include <io.h>
+#include <windows.h>
```

Comment:
Probably a good idea to add:

```
#define WIN32_LEAN_AND_MEAN
#define VC_EXTRA_LEAN
#define NOMINMAX
```

before the `windows.h` include to avoid a ton of `Windows.h` cruft.  It particularly helps clang.